presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -763,6 +764,35 @@ public TableStatistics getTableStatistics(ConnectorSession session, JdbcTableHan
         return TableStatistics.empty();
     }
 
+    @Override
+    public void createSchema(JdbcIdentity identity, String schemaName)
+    {
+        String sql = "CREATE SCHEMA " + quoted(schemaName);
+
+        try (Connection connection = connectionFactory.openConnection(identity)) {
+            execute(connection, sql);
+        }
+        catch (SQLException e) {
+            throw new PrestoException(JDBC_ERROR, e);
+        }
+    }
+
+    @Override
+    public void dropSchema(JdbcIdentity identity, String schemaName)
+    {
+        if (!getTableNames(identity, Optional.of(schemaName)).isEmpty()) {
+            throw new PrestoException(SCHEMA_NOT_EMPTY, "Schema not empty: " + schemaName);
```

Comment:
I am not convinced we need this check.
Even if we do the check, a table might be created concurrently (between now and when we issue `DROP SCHEMA`).

What if we do not do this check at all?
I know we won't produce as good err msg, but instead we will produce always the same message (from the underlying database).

cc @electrum